exporter/uptraceexporter/factory.go

```diff
@@ -74,5 +71,6 @@ func createTracesExporter(
 		exporterhelper.WithTimeout(exporterhelper.TimeoutSettings{Timeout: 0}),
 		exporterhelper.WithRetry(oCfg.RetrySettings),
 		exporterhelper.WithQueue(oCfg.QueueSettings),
+		exporterhelper.WithStart(exporter.Start),
 		exporterhelper.WithShutdown(exporter.Shutdown))
```

Comment:
same the Shutdown can be `shutdown`

Reply:
Agreed. I did not want to change their existing implementation when doing this PR, just wanted the minimal changes needed. I will happy change `Shutdown` too.